### ISSUE TITLE
ux: lockpad is far too distracting

### ISSUE THREAD

**Issue by cyphar** [vector-im/riot-android#820]

**Comment by ylecollen** [comment]
Do you talk about about the room activity ? the one near the edit text ?

**Comment by cyphar** [comment]
Yes. Not to mention that once encryption is enabled for a room it can't be disabled, there must be some nicer way of presenting "this is encrypted".

**Comment by Matrixcoffee** [comment]
It's not really too big per say, but it looks pretty cramped between my own avatar and the "Type a message" text. One for the UI team I think.

**Comment by zecakeh** [comment]
How about putting it in front of the timestamp ?

**Comment by dkanada** [comment]
@bmarty What about showing the icon only if it is not the default state of the room? So if the room is encrypted only show the icon for unencrypted messages or unverified devices, and if the room is unencrypted only showing the icon for encrypted messages.

**Comment by bmarty** [comment]
@dkanada I think we are talking about this one, which is a little big when the lock is open:

![lock_icon](https://user-images.githubusercontent.com/3940906/44465091-d7865400-a61c-11e8-9bec-4076280096e6.png)

**Comment by dkanada** [comment]
Ah, my bad I didn't notice. What if that one was a badge on the user avatar right next to it, similar to the chat icons in the main activity? I think it would get the point across but it might also be inconsistent design. Another option is not using an icon at all and just keeping the hint text.

The design really depends on where the encryption feature is heading. In the room settings there is a note that encryption can be toggled in the future, is that still planned? Also, will users ever be able to send non-encrypted messages to an encrypted room and vice versa, or is matrix not set up that way?

Should I open a new issue for the message encryption status icon? I feel like the two are somewhat related.

**Comment by dkanada** [comment]
Another option is modifying the lock icon to be uniform for both states, like the android icons below. That is outside my scope though since I am guessing that would be a cross platform change.

![lock](https://user-images.githubusercontent.com/21353219/44468558-6ff1f800-a660-11e8-886b-5e090e14987c.png) ![lock-open](https://user-images.githubusercontent.com/21353219/44468562-72ece880-a660-11e8-9538-c7150f6941a4.png)

**Comment by bmarty** [comment]
A designer is working on the future version of Riot, I'm pretty sure the Room screen will be refreshed.

CC @nadonomy

**Comment by dkanada** [comment]
I wasn't aware that was happening. Will the matrix blog cover planned design changes as well or is there another place I can follow those discussions?

**Comment by danielfrankcom** [comment]
Are the mentioned future changes still going on? Is there somewhere that the progress or the plan itself can be viewed? Thanks.
